### DISCUSSION TITLE
Beautiful edges in Microsoft and other professional

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#39] [Ideas & Feature Requests]
Microsoft Whiteboard for example has these beautiful edges when drawing fast at the end. I think it would be nice to implement if it is achievable.
![SmartSelect_20230925_173516_Whiteboard.jpg](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/1dee6289-4bd7-4b11-88bb-8d3730f1226a)

That works with input stabilization rather good, though not perfect

**Comment by personalizedrefrigerator** [comment]
As commented on the selection handles discussion, I hope to implement this soon, but for reference, the work done so far (which is **very** incomplete) can be found [on this branch](https://github.com/personalizedrefrigerator/js-draw/tree/work/pressure-inference).
